src/Aspire.Dashboard/Components/Controls/ApplicationName.razor

```diff
@@ -0,0 +1,54 @@
+﻿@namespace Aspire.Dashboard.Components
+@using Aspire.Dashboard.Model
+@using System.Globalization
+@inject IDashboardClient DashboardClient
+
+@code {
+    [Parameter]
+    public required string ResourceName { get; init; }
+
+    [Parameter]
+    public required IStringLocalizer Loc { get; init; }
+
+    private string? applicationName;
+
+    protected override void OnInitialized()
+    {
+        SetApplicationName();
+    }
+
+    protected override void OnParametersSet()
+    {
+        SetApplicationName();
+    }
+
+    private void SetApplicationName()
```

Comment:
Why isn't this component using async await and OnInitializedAsync?

Reply:
Oh awesome, I didn't see that. Thanks. Much nicer.